Ginger/GingerCoreNET/GingerCoreNET.csproj

```diff
@@ -544,6 +546,9 @@
     <None Update="Images\%40ItemPrev.jpg">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="NewSelfHealing\SelfHealingPropertyWeightage.json">
```

Comment:
keep the folder name as SelfHealingConfig

Reply:
done